Code/Core/FileUtils.cpp

```diff
@@ -0,0 +1,132 @@
+#include "sadpch.h"
+
+#include "FileUtils.h"
+#include <stdlib.h>
+#include <fstream>
+#include <sstream>
+#include <filesystem>
```

Comment:
Now that I think of it, I'm pretty okay if we move these 3 C++ includes into `sadpch.h` under `#include <algorithm>`.

I think these will mostly be used in these/similar classes anyways.

Also, if this is completed we can remove the `fstream` include on line 5 of `ConfigManager.cpp`.